<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lab_undone">
        <property name="text">
         <string>未完成文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="ptet_undone"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lab_finish">
        <property name="text">
         <string>完成文件  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="ptet_finish"/>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbox_finish">
        <property name="text">
         <string>完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbox_undone">
        <property name="text">
         <string>未完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save_data">
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lab_file_directory">
        <property name="text">
         <string>文件根目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="let_file_directory">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
